addons/analytic/models/analytic_line.py

```diff
@@ -11,6 +11,7 @@ class AccountAnalyticLine(models.Model):
     _description = 'Analytic Line'
     _order = 'date desc, id desc'
     _check_company_auto = True
+    _check_company_domain = models.check_company_domain_parent_of
```

Comment:
Shouldn't touch this, am I missing something?

Reply:
forgot to take it off